Lib/Cookie.py

```diff
@@ -226,6 +228,8 @@
 #
 # Define an exception visible to External modules
```

Comment:
I do not quite like this particular change. Maybe the right way to change this is to write this as a docstring...